doc/access.md

```diff
@@ -22,6 +22,10 @@ of members is [here][Build WG Members].
 A subsection of build members have access to infra machines
 (`infra-*`). The list of members is [here][Infra Admins].
 
+The infra group also have access to:
```

Comment:
have --> has

Reply:
Good nit! I think a group can be singular or plural though ([SO answer](https://english.stackexchange.com/questions/39838/is-group-singular-or-plural)).